src/graphql/search/filters/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { Filter, FilterOptions } from '../../../generated/graphql'
+
+// Combine both the filters and return the filters
+export function combineFilters(filters: Array<Filter>): Array<Filter> {
+  return filters.reduce((acc: any, cur: any) => {
+    // Find Filters that have the same type
+    const matching = acc.find((a: Filter) => a.type === cur.type)
+
+    if (matching) {
+      // Next to a matching type, the Filters can also have options with matching IDs
+      const options = [...matching.options, ...cur.options].reduce((acc: any, cur: any) => {
```

Comment:
Same here, also a bit confusing that this scope has `acc` and `cur` which already exist in the scope above it.